R/submission.R

```diff
@@ -0,0 +1,25 @@
+#' Wraps challengeutils.utils.change_submission_status. Written for tests
+#'
+#' @param submissionid Id of a submission
+#' @param status Change a submision to this status. Default to RECEIVED.
+.change_submission_status <- function(submissionid, status = "RECEIVED") {
+  change_status <- challengeutils$utils$change_submission_status
+  change_status(syn = syn, submissionid = submissionid, status = status)
+}
+
+#' Changes a status of a submission
+#'
+#' @param submissionid Id of a submission
+#' @param status Change a submision to this status. Default to RECEIVED.
```

Comment:
obviously i guess you can set this to anything but should this link to docs that indicate what the status options are and what they mean?